deploy/helm/sumologic/values.yaml

```diff
@@ -5098,15 +5098,21 @@ tailing-sidecar-operator:
   scc:
     create: false
 
-## Configure OpenTelemetry Operator - Tracing
+## Configure OpenTelemetry Operator - Instrumentation
 ## ref: https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-operator
 opentelemetry-operator:
   enabled: false
 
-  ## Specific for Tracing - Instrumentation resource creation
+  ## Specific for Sumo Logic chart - Instrumentation resource creation
+  instrumentationJobImage:
```

Comment:
please all information about new parameters to https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/README.md

Reply:
Thanks for catch Kasia I will create a separated PR for that :)